meta-mender-toradex-nxp/recipes-bsp/u-boot/u-boot-toradex-fsl-fw-utils_%.bbappend

```diff
@@ -0,0 +1,4 @@
+# We provide our custom fw_env.config and that is why we include
+# u-boot-mender-common instead of u-boot-fw-utils-mender
+require recipes-bsp/u-boot/u-boot-mender-common.inc
```

Comment:
I believe we covered this on the mailing list extensively [1].

[1]. https://groups.google.com/a/lists.mender.io/d/msg/mender/1vxZYPT0ysc/P9WJc4KZAwAJ